### ISSUE TITLE
Think of a better name...?

### ISSUE THREAD

**Issue by ircmaxell** [ircmaxell/php-compiler#57]
PHP-Compiler seems too blah. Perhaps rename to PHPPHP? Or something else more creative?

**Comment by BackEndTea** [comment]
How about PPHP? 
PHP PHP Hypertext Preprocessor

**Comment by driusan** [comment]
phpc? (I'm not very good at naming either.)

**Comment by WyriHaximus** [comment]
PHP²

**Comment by azjezz** [comment]
PAC - PHP AOT Compiler

**Comment by optiman** [comment]
PHC - PHP HyperCompiler

**Comment by gren236** [comment]
ComPHiler?

**Comment by someniatko** [comment]
PHPHP

**Comment by wujunze** [comment]
PHPHP  +1

**Comment by tommyknocker** [comment]
phpc +1

**Comment by na2axl** [comment]
Or why not a name which is not directly related to the  scope of the project (and then avoid strange strings with a lot of P and H :yum:)...

I'm proposing **Elephant** with a little slogan like *The PHP Compiler*.

It sounds more elegant to me (like other names, Laravel or Composer to list a few).

Beside this proposition, +1 for ComPHiler by @gren236

**Comment by rkyoku** [comment]
@na2axl I agree that, even though `phpc` reminds me of the old days (e.g. `gcc`) and is therefore a nice one, it lacks however... **Branding**. `Elephant` is clean and winks away at the PHP logo. It's a good find!

**Comment by PurHur** [comment]
I want to make an argument against "Elephant".
We want to make the compiled php as our target. This will save a lot of resources (CO2 and Energy) if we push "php compiled" as a brand. 
The future of this project should be that the Wordpress project makes a Issue "Lets go php compiled"
